includes/reader-activation/sync/class-esp-sync.php

```diff
@@ -108,6 +109,11 @@ public static function sync( $contact, $context = '' ) {
 		 */
 		$contact = \apply_filters( 'newspack_esp_sync_contact', $contact, $context );
 
+		if ( ! empty( $contact['metadata']['registration_date'] ) ) {
```

Comment:
I think it's fine if we did this transformation where this is first created [here](https://github.com/Automattic/newspack-plugin/blob/954e01b792d425f670f415fc4d3142e1b4bcb8ba/includes/reader-activation/sync/class-woocommerce.php#L350), so it's together with all the others and we assure that that method alaready output all the dates in the right timezone